download-libzim.js

```diff
@@ -56,4 +62,9 @@ for (let url of urls) {
         });
 }
 
-exec(`ln -s lib/x86_64-linux-gnu/libzim.so.6 download/libzim.so.6`);
\ No newline at end of file
+if (os.type() === 'Linux') {
+    exec(`ln -s lib/x86_64-linux-gnu/libzim.so.6 download/libzim.so.6`);
+} else if (os.type() === 'Darwin') {
+    // exec(`ln -s download/lib/libzim.6.dylib /usr/local/lib/libzim.6.dylib`);
+    exec(`cp download/lib/libzim.6.dylib /usr/local/lib`)
```

Comment:
I tried creating a symlink for libzim, but I was getting this exception while running test

```
dlopen(/Users/Utkarsha/Documents/Projects/node-libzim/build/Release/zim_binding.node, 1): Library not loaded: libzim.6.dylib
     Referenced from: /Users/Utkarsha/Documents/Projects/node-libzim/build/Release/zim_binding.node
     Reason: image not found
```
So, I directly copied the dynamic library to `/usr/local/lib/libzim.6.dylib`

Reply:
You should not have to do that.